### ISSUE TITLE
What additional features to add?

### ISSUE THREAD

**Issue by bahmutov** [bahmutov/generator-node-bahmutov#14]
A couple of ideas for additional tools to be included in this generator

- [space-hogs](https://github.com/dylang/space-hogs#readme) for measuring large folders, maybe ignoring the `node_modules`

**Comment by jimthedev** [comment]
* nodemon instructions for live-reload. Something like `nodemon --exec "npm run docker-full-restart"`

**Comment by bahmutov** [comment]
hmm, interesting @jimthedev - I just don't want to install nodemon as dev dependency

**Comment by jimthedev** [comment]
Agree. I think this could instruct to have nodemon globally. The main thing here is just to show that watching w/docker is possible but requires a specific command.
